### ISSUE TITLE
Assertion Failure When ThenStmt in IfInjectionPoint is not a BlockStmt

### ISSUE THREAD

**Issue by jonathanmetzman** [google/graphicsfuzz#196]
I'll try to take a look at what is going on here and/or find a better reproducer (this one requires 4 variants and happens on the 4th).

To reproduce (assuming `~/Downloads/shaders` is the shaders directory you shared on drive), using prebuilt binaries from [v1.1](https://github.com/google/graphicsfuzz/releases/download/v1.1/graphicsfuzz.zip):
```
$ java -ea -cp jar/tool-1.0.jar com.graphicsfuzz.generator.tool.GenerateShaderFamily ~/Downloads/shaders/references/GLSL_300_es/33113.0.json ~/Downloads/shaders/donors "300 es" /tmp/output --seed 400758670 --webgl --aggressively-complicate-control-flow --num-variants 4 --verbose

09:54:54.321 INFO  GenerateShaderFamily - Trying variant 4 (produced 2 of 4)
09:54:54.321 INFO  GenerateShaderFamily - Generating variant with inner seed 1091605847
java.lang.AssertionError
        at com.graphicsfuzz.generator.transformation.injection.IfInjectionPoint.inject(IfInjectionPoint.java:42)
        at com.graphicsfuzz.generator.transformation.controlflow.AddJumpStmts.apply(AddJumpStmts.java:55)
        at com.graphicsfuzz.generator.tool.Generate.applyControlFlowComplication(Generate.java:558)
        at com.graphicsfuzz.generator.tool.Generate.transformShader(Generate.java:264)
        at com.graphicsfuzz.generator.tool.Generate.generateVariant(Generate.java:185)
        at com.graphicsfuzz.generator.tool.Generate.generateVariant(Generate.java:219)
        at com.graphicsfuzz.generator.tool.GenerateShaderFamily.mainHelper(GenerateShaderFamily.java:216)
        at com.graphicsfuzz.generator.tool.GenerateShaderFamily.main(GenerateShaderFamily.java:356)
```

**Comment by afd** [comment]
I can reproduce the issue.

**Comment by afd** [comment]
Can be reproduced more directly with:

java -ea -cp jar/tool-1.0.jar com.graphicsfuzz.generator.tool.Generate ~/Downloads/shaders/references/GLSL_300_es/33113.0.json ~/Downloads/shaders/donors "300 es" output.json --webgl --seed 1091605847 --aggressively-complicate-control-flow

**Comment by afd** [comment]
Fixed by b9a52407d248f8ef71ef54617a2069907e7034b2.
